Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -64,10 +64,12 @@ function vtkOpenGLImageMapper(publicAPI, model) {
       model.openGLCamera = model._openGLRenderer.getViewNodeFor(
         ren.getActiveCamera()
       );
+      /*
       // is slice set by the camera
       if (model.renderable.getSliceAtFocalPoint()) {
         model.renderable.setSliceFromCamera(ren.getActiveCamera());
       }
+      */
```

Comment:
I have removed/disabled this as it feels like an interaction setup, and shouldn't be part of a mapper. Ideally a backend-mapper shouldn't modify user/app facing properties of a core mapper like this. Such events can be easily setup using application side callbacks, etc.